languages/en.json

```diff
@@ -0,0 +1,126 @@
+{
+	"header": {
+		"languages": {
+			"en": "English",
+			"de": "German"
+		},
+		"logout": "Logout",
+		"search": "Search trips..."
+	},
+	"footer": {
+		"followUs": {
+			"title": "Follow us"
+		},
+		"legal": {
+			"title": "Legal",
+			"imprint": "Imprint"
+		}
+	},
+	"forms": {
+		"signup": {
+			"title": "Sign up",
+			"emailStep": {
+				"title": "Select your email",
+				"invalidEmail": "Please provide a valid email",
+				"validEmail": "Email is valid",
+				"inititalValidation": "Please provide an email address to check availability",
+				"ongoingValidation": "Checking availability...",
+				"available": "This email is available",
+				"unavailable": "This email is already taken",
+				"nextStep": "Select your username"
+			},
+			"usernameStep": {
+				"title": "Select your username",
+				"invalidUsername": "Username must be between 4 and 15 characters",
+				"validUsername": "Username is valid",
+				"inititalValidation": "Please provide a valid username to check availability",
+				"ongoingValidation": "Checking availability...",
+				"available": "This username is available",
+				"unavailable": "This username is already taken",
+				"nextStep": "Select your password"
+			},
+			"passwordStep": {
+				"title": "Select your password",
+				"invalidPassword": "Password must be at least 8 characters long",
+				"validPassword": "Minimum length fulfilled",
+				"passwordsDontMatch": "Passwords don't match",
+				"passwordsMatch": "Passwords match",
+				"nextStep": "Register"
+			},
+			"tokenStep": {
+				"title": "Confirm your email",
+				"description": "Please enter the token you received via email!",
+				"initialValidation": "Enter the token to start validation",
+				"ongoingValidation": "Validating...",
+				"receiveCodeMessage": "Didn't receive a code?",
+				"resendCode": "Get a new one",
+			}
```

Comment:
Here is a missing comma. I wasn't able to start the server because of this
```suggestion
			},
```

Reply:
Ich mach gerade eh noch paar Übersetzungen, inzwischen funktioniert es wieder. Bin schon bei 170 Zeilen in der Übersetzungs-JSON. :D Die Error-Nachrichten mach ich auch noch und dann ist fürs erste gut